db/csv.go

```diff
@@ -0,0 +1,118 @@
+package db
```

Comment:
Not sure this package should be called `db` rather than `repository`, `ingest` or something more reflective of its contents.